src/burn/engine/plan.cpp

```diff
@@ -2742,6 +2764,84 @@ static HRESULT FinalizeSlipstreamPatchActions(
     return hr;
 }
 
+static HRESULT PlanDependencyActions(
```

Comment:
I'm waiting for mine to be merged as well. I confused as to what you're requesting be pulled when much of it is original in my PR. We've been using that code in production for about a year now.